### ISSUE TITLE
Win10 1909 - 10.0.18362.836

### ISSUE THREAD

**Issue by ghost** [stascorp/rdpwrap#1083]
Hi All

I am running the above build, and I've searched the issues to see if anyone has a solution, but i cannot find one.

![image](https://user-images.githubusercontent.com/66644442/84074659-cca3db80-a9ca-11ea-9ac0-647352a4234d.png)

Installed RDPWrap today, and get the following:
![image](https://user-images.githubusercontent.com/66644442/84074713-dc232480-a9ca-11ea-916a-0a5054a2ffee.png)

I assume its not listening and not supported because the versions do not match?

Can anyone help? 

Thanks

**Comment by romanad** [comment]
I confirm the issue is urgent.

**Comment by leonarriagada** [comment]
[10.0.18362.836]
LocalOnlyPatch.x86=1
LocalOnlyOffset.x86=B7D06
LocalOnlyCode.x86=jmpshort
LocalOnlyPatch.x64=1
LocalOnlyOffset.x64=82FC5
LocalOnlyCode.x64=jmpshort
SingleUserPatch.x86=1
SingleUserOffset.x86=50515
SingleUserCode.x86=nop
SingleUserPatch.x64=1
SingleUserOffset.x64=DBFC
SingleUserCode.x64=Zero
DefPolicyPatch.x86=1
DefPolicyOffset.x86=50249
DefPolicyCode.x86=CDefPolicy_Query_eax_ecx
DefPolicyPatch.x64=1
DefPolicyOffset.x64=1FE15
DefPolicyCode.x64=CDefPolicy_Query_eax_rcx
SLInitHook.x86=1
SLInitOffset.x86=5A75A
SLInitFunc.x86=New_CSLQuery_Initialize
SLInitHook.x64=1
SLInitOffset.x64=22DDC
SLInitFunc.x64=New_CSLQuery_Initialize

[10.0.18362.836-SLInit]
bInitialized.x86      =D577C
bServerSku.x86        =D5780
lMaxUserSessions.x86  =D5784
bAppServerAllowed.x86 =D578C
bRemoteConnAllowed.x86=D5790
bMultimonAllowed.x86  =D5794
ulMaxDebugSessions.x86=D5798
bFUSEnabled.x86       =D579C

bInitialized.x64      =F6A8C
bServerSku.x64        =F6A90
lMaxUserSessions.x64  =F6A94
bAppServerAllowed.x64 =F6A9C
bRemoteConnAllowed.x64=F6AA0
bMultimonAllowed.x64  =F6AA4
ulMaxDebugSessions.x64=F6AA8
bFUSEnabled.x64       =F6AAC

![Anotación 2020-06-08 231706](https://user-images.githubusercontent.com/15932209/84102364-40031880-a9de-11ea-86f3-2906f9835791.png)

Pero aún así no puedo conectarme y no se que será, se me ocurre que la versión es diferente. 10.0.18362.836 en cambio en winver aparece 10.0.18363.836

**Comment by ghost** [comment]
Where exactly do I paste the configuration information above in the .ini file?

**Comment by maximallist** [comment]
Hi! The computer in the domain. Remote access is enabled. When you try to connect as a second user, a message appears: "The number of allowed connections to this computer is limited ..." and the connection fails.
![изображение](https://user-images.githubusercontent.com/8707151/84113431-fd742700-aa43-11ea-9f1b-bb023cfca1b1.png)

**Comment by robbak** [comment]
You have posted this message in someone else's thread. Please start a new
thread if you want assistance.

**Comment by robbak** [comment]
Just at the end of the existing .ini file. 

Or pull the .ini file provided by @affinityv in comment #1022

**Comment by S474N** [comment]
Have 10.0.18362.836 in app, but version of Windows is Win 10 Home, 1909 build 18363.900

Service state: Stopped
Listeren state: Not listening

**Comment by leonarriagada** [comment]
Yo tuve que usar otra versión del .ini
Encontré en estos datos que adjunto en el txt. Espero les sirva, saludos
[rdpwrap.txt](https://github.com/stascorp/rdpwrap/files/4760520/rdpwrap.txt)

**Comment by gaoyanhui** [comment]
> Have 10.0.18362.836 in app, but version of Windows is Win 10 Home, 1909 build 18363.900
> 
> Service state: Stopped
> Listeren state: Not listening

hey, Has your problem been solved

**Comment by S474N** [comment]
Don't know, because I need it immediately. So i bought Win 10 Pro key and upgraded to Pro ;)

**Comment by dybrn** [comment]
it did not work for me.
![image](https://user-images.githubusercontent.com/2583051/84539517-66f46f80-aca8-11ea-8c59-62040d440c33.png)

**Comment by dybrn** [comment]
Yes this worked. (had to update the correct INI file)
![image](https://user-images.githubusercontent.com/2583051/84539758-df5b3080-aca8-11ea-9531-1219026cdc14.png)

**Comment by fw-media** [comment]
I wrote an update based on Version 657.

[rdpwrap_ini_updater_836.zip](https://github.com/stascorp/rdpwrap/files/4820952/rdpwrap_ini_updater_836.zip)

**Comment by jairhinestroza1** [comment]
Hola, la conexión funciona correctamente, pero a los 10 minutos de inactividad me desconecta y cierra la sesión de escritorio remoto del usuario que estoy usando en el servidor perdiendo la información como documentos que no haya guardado, a que se puede deber esto?
De ante mano muchas gracias por la ayuda.

**Comment by affinityv** [comment]
It's not just a disconnect with you able to reconnect the same session after without data loss?

**Comment by jairhinestroza1** [comment]
No, cuando me desconecto se cierra la sesión del servidor en la que estoy trabajando y cuando quiero volver a ingresar me abre la sesión con todos los programas que estaba usando cerrados y archivos que no he guardado borrados, como si nunca hubiera entrado.

**Comment by affinityv** [comment]
Not a new session with the old one still being listed in task manager?

**Comment by iofirag** [comment]
> [10.0.18362.836]
> LocalOnlyPatch.x86=1
> LocalOnlyOffset.x86=B7D06
> LocalOnlyCode.x86=jmpshort
> LocalOnlyPatch.x64=1
> LocalOnlyOffset.x64=82FC5
> LocalOnlyCode.x64=jmpshort
> SingleUserPatch.x86=1
> SingleUserOffset.x86=50515
> SingleUserCode.x86=nop
> SingleUserPatch.x64=1
> SingleUserOffset.x64=DBFC
> SingleUserCode.x64=Zero
> DefPolicyPatch.x86=1
> DefPolicyOffset.x86=50249
> DefPolicyCode.x86=CDefPolicy_Query_eax_ecx
> DefPolicyPatch.x64=1
> DefPolicyOffset.x64=1FE15
> DefPolicyCode.x64=CDefPolicy_Query_eax_rcx
> SLInitHook.x86=1
> SLInitOffset.x86=5A75A
> SLInitFunc.x86=New_CSLQuery_Initialize
> SLInitHook.x64=1
> SLInitOffset.x64=22DDC
> SLInitFunc.x64=New_CSLQuery_Initialize
> 
> [10.0.18362.836-SLInit]
> bInitialized.x86 =D577C
> bServerSku.x86 =D5780
> lMaxUserSessions.x86 =D5784
> bAppServerAllowed.x86 =D578C
> bRemoteConnAllowed.x86=D5790
> bMultimonAllowed.x86 =D5794
> ulMaxDebugSessions.x86=D5798
> bFUSEnabled.x86 =D579C
> 
> bInitialized.x64 =F6A8C
> bServerSku.x64 =F6A90
> lMaxUserSessions.x64 =F6A94
> bAppServerAllowed.x64 =F6A9C
> bRemoteConnAllowed.x64=F6AA0
> bMultimonAllowed.x64 =F6AA4
> ulMaxDebugSessions.x64=F6AA8
> bFUSEnabled.x64 =F6AAC
> 
> ![Anotación 2020-06-08 231706](https://user-images.githubusercontent.com/15932209/84102364-40031880-a9de-11ea-86f3-2906f9835791.png)
> 
> Pero aún así no puedo conectarme y no se que será, se me ocurre que la versión es diferente. 10.0.18362.836 en cambio en winver aparece 10.0.18363.836

Thanks! save my day.
tip for others:
- run RDPConf.exe and get the exact version. (10.0.XXXX.........)
- google for this compatible .ini values for this version (10.0.XXXX.........)
after found it.

- run install script
- add installed folder to exclusion folder list in windows defender / anti-virus
- disconnect from internet
- edit .ini file, and append this lines at the end.
- connect to internet
- run RDPConf.exe again (all should be green now)

**Comment by jairhinestroza1** [comment]
No, se abre una nuev

> Not a new session with the old one still being listed in task manager?

No, se abre una nueva sin ninguna información de la anterior.
Mi versión de windows 10 es 19041

**Comment by 7BigD** [comment]
> Have 10.0.18362.836 in app, but version of Windows is Win 10 Home, 1909 build 18363.900
> 
> Service state: Stopped
> Listeren state: Not listening
I have the same question. I am finding the solution. 😫

**Comment by strarsis** [comment]
Was able to fix it using some INI from another user - but `RDPConf.exe` incorrectly indicates an unsupported version, see https://github.com/stascorp/rdpwrap/issues/1054#issuecomment-742637185.

**Comment by fragtion** [comment]
Need support for 10.0.19045.2673 also
